utils/sql/svn/1392_recipe_learning.sql

```diff
@@ -9,4 +9,5 @@ alter table `tradeskill_recipe` add column `must_learn` tinyint not null default
 
 insert into rule_values (ruleset_id, rule_name, rule_value, notes) values
 (1, 'Skills:UseLimitTradeskillSearchSkillDiff', 'true', 'Enables the limit for the maximum difference between trivial and skill for recipe searches and favorites.'),
-(1, 'Skills:MaxTradeskillSearchSkillDiff', '50', 'The maximum difference in skill between the trivial of an item and the skill of the player if the trivial is higher than the skill. Recipes that have not been learnt or made at least once via the Experiment mode will be removed from searches based on this criteria.');
\ No newline at end of file
+(1, 'Skills:TrivialNoFail', 'false', 'Makes all trivial recipes no-fail.'),
```

Comment:
Adding this here is unnecessary, rules are handled automatically by the source.

Reply:
So that SQL file is basically deprecated then?

Reply:
Yes, 100%, it's from SVN (subversion) days which is over 10 years old at this point, long before this repository existed.

Reply:
We automatically add new rules in `rulesys.cpp`.
```cpp
	for (const auto &d : rule_data) {
		if (std::find(v.begin(), v.end(), d.first) == v.end()) {
			injected_rule_entries.push_back(
				std::tuple<int, std::string, std::string, std::string>(
					rule_set_id,
					d.first,
					d.second.first,
					Strings::Escape(*d.second.second)
				)
			);

			if (!quiet_update) {
				LogInfo(
					"Adding new rule [{}] ruleset [{}] ({}) value [{}]",
					d.first,
					rule_set_name,
					rule_set_id,
					d.second.first
				);
			}
		}
	}
	```